app.py

```diff
@@ -0,0 +1,10 @@
+from flask import Flask, jsonify
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return jsonify({"Application Name": "X-Ray Scan using ML", "Version": "1.0", "Type":"REST"})
```

Comment:
Have you done unit and integration test for this. If yes then provide link .
Thank you for the contribution.

Reply:
I will add `unit-test` into the application, can you help me with the basic workflow of the application ?
I will add a temporary-frontend as well.
please add labels shown at the right hand side, as `hacktoberfest` and `hacktober-fest`.
Thanks